install.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+set -eu
+
+CHANNEL=${CHANNEL:-alpha}
+VERSION=${VERSION:-v0.12.0}
+INSTALLPATH=${INSTALLPATH:-"/usr/local/bin/kubectl-crossplane"}
+
+os=$(uname -s)
+arch=$(uname -m)
+OS=${OS:-"${os}"}
+ARCH=${ARCH:-"${arch}"}
+OS_ARCH=""
+
+BIN=${BIN:-crank}
+
+unsupported_arch() {
+  local os=$1
+  local arch=$2
+  echo "Crossplane does not support $os / $arch at this time."
+  exit 1
+}
+
+case $OS in
+  CYGWIN* | MINGW64*)
+    if [ $ARCH == "amd64" ]
+    then
+      os_arch=windows_amd64
+      BIN=crank.exe
+    else
+      unsupported_arch $OS $ARCH
+    fi
+    ;;
+  Darwin)
+    os_arch=darwin_amd64
+    ;;
+  Linux)
+    case $ARCH in
+      x86_64|amd64)
+        os_arch=linux_amd64
+        ;;
+      arm64)
+        os_arch=linux_arm64
+        ;;
+      *)
+        unsupported_arch $OS $ARCH
+        ;;
+    esac
+    ;;
+  *)
+    unsupported_arch $OS $ARCH
+    ;;
+esac
+
+url="https://releases.crossplane.io/${CHANNEL}/${VERSION}/bin/linux_amd64/${BIN}"
```

Comment:
```suggestion
url="https://releases.crossplane.io/${CHANNEL}/${VERSION}/bin/${os_arch}/${BIN}"
```

Shouldn't it be like this?

Reply:
Yep! Thanks for catching :)